.github/workflows/ci.yml

```diff
@@ -45,11 +45,11 @@ jobs:
           npm run lint:deps
 
       - name: Add nonce for parallel tests
-        run: echo ::set-env name=PERCY_PARALLEL_NONCE::`date +%s`
+        run: echo "PERCY_PARALLEL_NONCE=`date +%s`" >> $GITHUB_ENV
 
-      - uses: percy/exec-action@v0.3.0
-        with:
-          command: npm run test-coverage
+      - name: Run percy exec
+        if: github.repository == 'rust-lang/crates.io'
```

Comment:
why do you want to run this only on this repo? wouldn't that mean that it doesn't run for PRs coming from forked repos?

Reply:
Note that this rejects any builds on forked repo, not here. I usually run CI on my fork before submitting PR to make sure things but it seems its CI triggers percy, see https://github.com/JohnTitor/crates.io/pull/17 and https://percy.io/crates-io/crates.io/builds/7205884. I think it's better not to trigger the check in such a case.

Reply:
hmm, yeah, I'm surprised it still ran on this PR, but maybe I misunderstood how GitHub does things 😅 

I'm wondering, should we still run `npm run test-coverage` if `if: github.repository != 'rust-lang/crates.io'`?

Reply:
`if: github.repository == 'rust-lang/crates.io'` means that "if the org name is `rust-lang` and the repo name is `crates.io`, run the below command" so this should work as expected. `if: github.repository != 'rust-lang/crates.io'` won't trigger percy here :P

Reply:
> so this should work as expected

yeah, but I didn't expect that condition to be true for a pull request CI job from a forked repository, like for this PR

> if: github.repository != 'rust-lang/crates.io' won't trigger percy here

exactly, but it not only does not trigger Percy currently, it doesn't run the test suite at all anymore, which seems bad, no?

Reply:
> yeah, but I didn't expect that condition to be true for a pull request CI job from a forked repository, like for this PR

Nah, this PR is opened on **this repository** so GHA is triggered here, not on forked repo. If a PR is opened on a forked repo, this doesn't trigger percy.

Reply:
> I'm wondering, should we still run `npm run test-coverage` if `if: github.repository != 'rust-lang/crates.io'`?

Yeah, I think so. It seems like we should run these tests even when not triggering percy. I'm not as worried about the travis config, since I think we're close to gating on GH Actions.

Reply:
Okay, addressed via c8fa8ff.